### PR TITLE
Feature/change port status fix

### PR DESCRIPTION
**Pull request by Settis**
Fix merge for pull request #1080

### CODE REVIEW

File: services/src/atdd/src/test/resources/features/mvp.1/SwitchPortConfiguration.feature [hunk 1]
```diff
@@ -4,12 +4,12 @@ Feature: Configure switch port
   Background:
     Given a clean controller
     And created simple topology from two switches
-    And port status for switch "00:01:00:00:00:00:00:01" port "2" is 'Up'
+    And all port statuses for switch "00:01:00:00:00:00:00:01" are 'Up'
 
   Scenario: Change port status
     When change port status to 'Down' for switch "00:01:00:00:00:00:00:01" port "2"
     Then port status for switch "00:01:00:00:00:00:00:01" port "2" is 'Down'
+    And all port statuses for switch "00:01:00:00:00:00:00:01" except for port "2" are 'Up'
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply services/src/atdd/src/test/resources/features/mvp.1/SwitchPortConfiguration.feature]: Which switch, other from which port? I am not going to 'save' the previous step results.

[the remark this answers is not in the document]
Comment (ylobankov) [reply services/src/atdd/src/test/resources/features/mvp.1/SwitchPortConfiguration.feature]: Sorry, when I wrote the comment I had outdated version of your PR.

[the remark this answers is not in the document]
Comment (ylobankov) [reply services/src/atdd/src/test/resources/features/mvp.1/SwitchPortConfiguration.feature]: Something like that:
And all ports on switch "00:01:00:00:00:00:00:01" except port "2" are in "Up" state

**Comment by surabujin** [comment]
It would be great to squash it down to 2 commit - feature itself and style check fixes.

**Pull request merged**